pkgs/by-name/tr/tracy/default.nix

```diff
@@ -0,0 +1,131 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+
+  cmake,
+  ninja,
+
+  capstone,
+  darwin,
+  dbus,
+  freetype,
+  glfw,
+  pkg-config,
+  tbb,
+
+  withWayland ? stdenv.isLinux,
+  libglvnd,
+  libxkbcommon,
+  wayland-protocols,
+  wayland,
+}:
+
+stdenv.mkDerivation rec {
+  pname = if withWayland then "tracy-wayland" else "tracy-glfw";
+  version = "0.11.0";
+
+  src = fetchFromGitHub {
+    owner = "wolfpld";
+    repo = "tracy";
+    rev = "v${version}";
+    sha256 = "sha256-wGKnmCpmG70Xx/+RfZhXTLSY38AcGrZK/jxPpePkXik=";
+  };
+
+  patches = lib.optional (
+    stdenv.isDarwin && lib.versionOlder stdenv.hostPlatform.darwinMinVersion "11"
+  ) ./dont-use-the-uniformtypeidentifiers-framework.patch;
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+    pkg-config
+  ] ++ lib.optionals stdenv.cc.isClang [ stdenv.cc.cc.libllvm ];
```

Comment:
Note that `gtk3` is another requirement on linux (https://github.com/wolfpld/tracy/pull/94).
```nix
++ lib.optionals stdenv.isLinux [ gtk3 ];
```